### PR TITLE
docs: first of several corrections to the scaling instructions

### PR DESCRIPTION
**Pull request by kimberlyxdavis**
## PR Checklist

- [ ] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [ ] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?

        modified:   docs/Kubernetes/kubernetes.md
        modified:   docs/assets/images/HighLevelArchitectureSID.png
        modified:   docs/assets/images/HighLevelArchitectureScaling.png

Corrected for general structure in anticipation of new instructions.

### CODE REVIEW

**Comment by matt-primrose** [comment]
@kimberlyxdavis merge conflicts have been resolved.

File: docs/Kubernetes/kubernetes.md [hunk 1]
```diff
@@ -1,12 +1,93 @@
...
+
+## Building the Docker Images
+1. Build and push the following images from the following open-amt-cloud-toolkit repositories MPS, RPS, and sample-web-ui into a public repository.
+```
+docker build . -t docker.io/vprodemo/mps:latest
+docker push docker.io/vprodemo/mps:latest
+```
+2. Update "docker.io/vprodemo" to the desired public repository and "mps" to the corresponding repository name. 
+
+### Prerequisites
```
Comment (rsdmike) [inline docs/Kubernetes/kubernetes.md:68]: these scripts no longer exists

File: docs/Kubernetes/kubernetes.md [hunk 2]
```diff
@@ -1,12 +1,93 @@
...
+**Install required software in Windows environment (in admin mode):**
+
+1. Run `\open-amt-cloud-toolkit\scripts\kubernetes\installchoco.bat` to install the choco package manager.
+2. Close and reopen the command window.
+3. Run `\open-amt-cloud-toolkit\scripts\kubernetes\installpackages.bat` to install the required packages.
+
+### Steps to Deploy
```
Comment (rsdmike) [inline docs/Kubernetes/kubernetes.md:75]: steps to deploy has changed.

**Review by rsdmike — CHANGES_REQUESTED** [review]
After building the docker images, i think linking to the two deployment options is the right move. Local or AKS

**Pull request closed without merging**